### DISCUSSION TITLE
Help opening model "Dealing with mixed and non-normative...."

### DISCUSSION THREAD

**Discussion by kmamor** [gama-platform/gama#206] [Q&A] [answered]
I read the paper 'Dealing with mixed and non-normative traffic. An agent-based simulation with the GAMA platform ' here [((https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0281658#:~:text=This%20model%2C%20developed%20in%20GAMA,can%20reproduce%20a%20credible%20traffic.))]

The authors  - some of whom I believe moderate this discussion kindly make their model available  here:  [(https://doi.org/10.6084/m9.figshare.21369090.v1)  ] .

I downloaded the files. Looking in the model folder none of the models open in GAMA despite being GAMA files. I get the following error message:
![error](https://github.com/gama-platform/gama/assets/45757038/3b609544-8f29-4693-8a82-48c93bb3e2b9)

If I click a GAMA file elsewhere that I have created from GAMA it simply opens itself within the GAMA environment. 

Any assistance would be appreciated. 

K

**Comment by kmamor** [comment]
I managed to open the file from within GAMA (file/open file)  which is all I need . However I still get an error message:

![error2](https://github.com/gama-platform/gama/assets/45757038/aae81dd1-e5e7-4181-bf7f-edf58543eff3)

K

**Reply by lesquoyb** [reply]
could you try in a new workspace ?
Else could you click on details and show us what is displayed ?

**Reply by kmamor** [reply]
I tried using new workspaces. When I try and open Traffic.gaml I get the following two error messages:

![error7](https://github.com/gama-platform/gama/assets/45757038/5cf7ec59-2a7e-4283-aec0-2f895666bc98)


![error8](https://github.com/gama-platform/gama/assets/45757038/2f89b350-3c18-4c88-8bc7-fc3c66ab5c63)


K

**Reply by lesquoyb** [reply, marked as answer]
ah ok I misunderstood your problem.
You need to create a new project in gama and then copy paste those files inside of it. Opening files without context (no project) is not recommended and is only a way to have a better text editor.

**Reply by kmamor** [reply]
That worked - thanks for your help. 


K
